qiskit/providers/aer/backends/wrappers/CMakeLists.txt

```diff
@@ -1,36 +1,16 @@
 include(Linter)
-
-find_package(PythonExtensions REQUIRED)
-find_package(PythonLibs REQUIRED)
```

Comment:
Move these two lines to the FindPybind11.cmake since pybind still needs the variables set by these modules (mainly PYTHON_INCLUDE_DIRS)

Reply:
They are already there: https://github.com/Qiskit/qiskit-aer/blob/master/cmake/FindPybind11.cmake#L17

Reply:
True! sorry, I meant move them up so that they're before the check